drivers/media/v4l2-core/v4l2-ctrls.c

```diff
@@ -2355,7 +2355,15 @@ void v4l2_ctrl_handler_free(struct v4l2_ctrl_handler *hdl)
 	if (hdl == NULL || hdl->buckets == NULL)
 		return;
 
-	if (!hdl->req_obj.req && !list_empty(&hdl->requests)) {
+	/*
```

Comment:
Commit text 

> (cherry picked from commit be7e8af98f3af729aa9f08b1053f9533a5cceb91)

Is that a partial cherry-pick, or complete?

Generally Phil will request for backports that the first line of the body is 
```Commit be7e8af98f3af729aa9f08b1053f9533a5cceb91 upstream```
or similar, so that it is obvious when rebasing on later trees that already include the patch.

(The patch does look very much like https://github.com/torvalds/linux/commit/be7e8af98f3af729aa9f08b1053f9533a5cceb91)

Reply:
This patch is a full cherry-pick from Hans' tree from before it made it to upstream. I think it was backported to 5.10 last night so may be redundant.

Reply:
"media: v4l2-ctrls: fix reference to freed memory" is in 5.10.36. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/log/?h=linux-5.10.y as https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=linux-5.10.y&id=5d0f6f5251a360f97dad0221ca60cf622caf17d2

"media: v4l2-ctrls: fix race condition in hdl->requests lists" doesn't appear to be there. Seeing  as the commit text didn't cc stable, it would have had to be a manual poke to get it backported.

Reply:
I got an (automated) email last night (presumably as I am listed as reporter and tester) giving as "[PATCH 5.10 315/530] media: v4l2-ctrls.c: fix race condition in hdl->requests list" and also patches against 5.11, 5.12, 5.4

Reply:
So should I just remove these two patches or what? The code will work without them but it will be crashy